jjb/anax/anax.yml

```diff
@@ -0,0 +1,21 @@
+- job:
+    name: jenkins-job
+    project-type: pipeline
+    defaults: global
```

Comment:
What is "defaults" defining here?

Reply:
we don't have defaults yet, but may be in the future. I remove it.